src/modules/deal-investor/dto/create-deal-investor-request.dto.ts

```diff
@@ -0,0 +1,67 @@
+import { ValidatedDateOnlyApiProperty } from '@ukef/decorators/validated-date-only-api-property.decorator';
+import { ValidatedStringApiProperty } from '@ukef/decorators/validated-string-api-property.decorator';
+import { DateString, UkefId } from '@ukef/helpers';
+
+export type CreateDealInvestorRequest = CreateDealInvestorRequestItem[];
+
+export class CreateDealInvestorRequestItem {
+  @ValidatedStringApiProperty({
+    description: 'The unique identifier of the portfolio associated to the business entity. The UKEF portfolio is E1.',
+    example: 'E1',
+    minLength: 0,
+    maxLength: 2,
+    required: false,
+    default: 'E1',
+  })
+  readonly portfolioIdentifier: string;
+
+  @ValidatedStringApiProperty({
+    description: "The unique identifier for the deal within the portfolio. It will be a 10-digit code beginning with either '0020', '0030', or '0040'.",
+    example: '0020900111',
+    length: 10, // TO-DO: check
+    pattern: /^00[2-4]0\d{6}$/,
+  })
+  readonly dealIdentifier: UkefId;
+
+  @ValidatedStringApiProperty({
+    description: 'The lender type code for the investor party of the deal.',
+    example: '500',
+    minLength: 0,
+    maxLength: 3,
+    required: false,
+    default: '500',
+  })
+  readonly lenderType?: string; // TO-DO: should this be numeric?
```

Comment:
Is the `lenderType` always numeric? If so, should we validate this?

Reply:
Looks like ACBS accepts this as numeric field on POST, but we always used to send it as string, so keeping it same is easier?

On GET ACBS returns this as string.

Reply:
Cool, thanks both, I've kept this the same